lesson03/assignment/index.js

```diff
@@ -0,0 +1,104 @@
+const localData = [
```

Comment:
You seem to have the data in two places, making your code slightly redundant. Both in your data.js file and hardcoded in here. Very small detail but thought I should point out.

Reply:
Thanks Tyler, I wasn't sure if changing the data.json to a data.js file and pulling it in via the script tag was incorrect, so I wanted to leave this option in as an alternative.  Thanks for the review!